runtime/src/main/java/org/corfudb/runtime/view/StreamOptionsBuilder.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.runtime.view;
+
+
+/**
+ * Created by maithem on 6/21/17.
+ */
+public class StreamOptionsBuilder {
+    private boolean ignoreTrimmed;
+
+    public StreamOptionsBuilder() {
```

Comment:
Ok, I guess one more nit, if we're going with the builder paradigm

StreamOptionsBuilder should just be a static class inside StreamBuilder
StreamOptions.builder() should give you a StreamOptionsBuilder

This is the paradigm that Guava follows (like ImmutableMap.builder().)